massdriver.yaml

```
@@ -3,29 +3,24 @@ name: gcp-gcs-bucket-static-assets
 description: "GCP Static Assets Bucket"
 source_url: github.com/massdriver-cloud/gcp-gcs-bucket-static-assets
 access: public
-type: bundle
+type: infrastructure
 
 params:
-  examples:
-    - __name: Example
-      cloud_dns_managed_zone:
-        name: my-foobar-zone
+  examples: []
   required:
-    - subdomain
-    - cloud_dns_managed_zone
+    - bucket
   properties:
-    subdomain:
-      type: string
-      title: Subdomain
-      minLength: 1
-      maxLength: 60
-      pattern: "^[a-z-*]+$"
-      message:
-        pattern: "Subdomain must be lowercase letters, numbers, or dashes"
-      description: Subdomain for the static bucket
-      default: assets
-    cloud_dns_managed_zone:
-      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/gcp-cloud-dns-managed-zone.json
+    bucket:
+      type: object
+      title: Bucket
+      required:
+        - force_destroy
+      properties:
+        force_destroy:
+          title: Enable force destroy of bucket
+          description: When enabled, the bucket and all objects within it are destroyed when the bundle is decommissioned.
+          default: true
```

```suggestion
          default: false
```
Probably should default this to false.